galsim/interpolant.py

```diff
@@ -126,6 +138,53 @@ def __ne__(self, other):
     def __hash__(self):
         return hash(repr(self))
 
+    def xval(self, x):
+        """Calculate the value of the interpolant kernel at one or more x values
+
+        Parameters
```

Comment:
This looks like numpy doc style, which is neither current doxygen nor future google style.  Just sayin...